sc/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotController.java

```diff
@@ -125,6 +140,21 @@ public String tSnePlotWithExpression(
         return GSON.toJson(model);
     }
 
+    @RequestMapping(
+            value = "json/experiments/{experimentAccession}/metadata",
+            method = RequestMethod.GET,
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ResponseBody
```

Comment:
You don't need the `@ResponseBody` annotation here (read about it here: https://www.baeldung.com/spring-request-response-body)

Reply:
Thanks! didn't know if we already have annotated the controller as `@RestController` then we don't need to have `@ResponseBody`

Will update the code.